changelog/changelog.py

```diff
@@ -307,15 +316,17 @@ def main():
         'generator': default_changelog_generator,
     },
     'spinalcordtoolbox': {
-        'labels': ['feature', 'documentation-internal', 'CI', 'bug', 'installation', 'documentation', 'enhancement', 'refactoring', 'git/github'],
+        'labels': ['feature', 'documentation-internal', 'CI', 'bug', 'installation', 'documentation', 'enhancement',
+                   'refactoring', 'git/github'],
         'generator': sct_changelog_generator,
     },
     'ivadomed': {
-        'labels': ['bug', 'dependencies', 'documentation', 'enhancement'],
+        'labels': ['feature', 'CI', 'bug', 'installation', 'documentation', 'dependencies', 'enhancement',
+                   'testing', 'refactoring'],
```

Comment:
Looking at this PR, I'm wondering if it might be better to get rid of these hardcoded labels entirely, and instead have the usage look like this:

```
changelog --repo ivadomed --labels feature bug installation documentation enhancement [...]
```

That way, `changelog` stays project-agnostic, and each project doesn't have to open a PR every time they make a change to their label scheme. :sweat_smile:

Reply:
that's not a bad idea. As long as the syntax is written somewhere for each project (for convenient copy/paste), that should do it

Reply:
Opened https://github.com/neuropoly/changelog/issues/29, so marking this as resolved.